backend/inspirehep/migrator/tasks.py

```diff
@@ -354,7 +354,7 @@ def process_references_in_records(uuids):
     try:
         for uuid in uuids:
             try:
-                record = InspireRecord.get_record(uuid)
+                record = InspireRecord.get_record(uuid, with_deleted=True)
```

Comment:
Do we need deleted?

Reply:
Yes, otherwise we will never upgrade all references properly during migration. (remember that until now we were always processing deleted records as this flag was not working before)